dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthorizationRestRepositoryIT.java

```diff
@@ -982,94 +986,153 @@ public void findByObjectAndFeatureTest() throws Exception {
         String adminToken = getAuthToken(admin.getEmail(), password);
 
         // verify that it works for administrators - with eperson parameter
-        getClient(adminToken).perform(get("/api/authz/authorizations/search/objectAndFeature")
+        getClient(adminToken).perform(get("/api/authz/authorizations/search/object")
                 .param("uri", comUri)
                 .param("projection", "level")
                 .param("embedLevelDepth", "1")
                 .param("feature", alwaysTrue.getName())
                 .param("eperson", admin.getID().toString()))
             .andExpect(status().isOk())
-            .andExpect(jsonPath("$.type", is("authorization")))
-            .andExpect(jsonPath("$._embedded.feature.id", is(alwaysTrue.getName())))
-            .andExpect(jsonPath("$.id", Matchers.is(admin.getID().toString() + "_" + alwaysTrue.getName() + "_"
-                    + comRest.getUniqueType() + "_" + comRest.getId())));
+            .andExpect(jsonPath("$.page.totalElements", is(1)))
+            .andExpect(jsonPath("$._embedded.authorizations", contains(
+                    allOf(
+                            hasJsonPath("$.id", is(admin.getID().toString() + "_" + alwaysTrue.getName() + "_"
+                                    + comRest.getUniqueType() + "_" + comRest.getId())),
+                            hasJsonPath("$.type", is("authorization")),
+                            hasJsonPath("$._embedded.feature.id", is(alwaysTrue.getName())),
+                            hasJsonPath("$._embedded.eperson.id", is(admin.getID().toString()))
+                    )
+            )));
 
         // verify that it works for administrators - without eperson parameter
-        getClient(adminToken).perform(get("/api/authz/authorizations/search/objectAndFeature")
+        getClient(adminToken).perform(get("/api/authz/authorizations/search/object")
```

Comment:
This looked odd to me at first, but you are correct :) We don't have an `/objectAndFeature` endpoint...it's named `/object` in the REST contract: https://github.com/DSpace/Rest7Contract/blob/main/authorizations.md#object